odo registry -o json (#5739)
redhat-developer/odo: docs/website/versioned_docs/version-3.0.0/command-reference/json-output.md

The `components` field lists the components either deployed in the cluster, or defined in the local Devfile.

> The `componentInDevfile` field gives the name of components present in the `components` list that are defined in the local Devfile, or is empty if no local Devfile is present.

Reviewer: I guess `components` should not be plural here, as we are now returning at most one name.

```suggestion
The `componentInDevfile` field gives the name of the component present in the `components` list and defined in the local Devfile, or is empty if no local Devfile is present.
```
